sources/editor/Stride.Core.Assets.Editor/View/DefaultPropertyTemplateProviders.xaml

```diff
@@ -807,29 +807,35 @@
-                           Minimum="{Binding Minimum, Converter={sd:Chained {cvt:DifferentValuesToNull}, {sd:ToDouble}}, FallbackValue=0}"
-                           Maximum="{Binding Maximum, Converter={sd:Chained {cvt:DifferentValuesToNull}, {sd:ToDouble}}, FallbackValue={x:Static s:Double.MaxValue}}"
-                           SmallChange="{Binding SmallStep, Converter={sd:Chained {cvt:DifferentValuesToNull}, {sd:ToDouble}}, FallbackValue=0.1}"
-                           LargeChange="{Binding LargeStep, Converter={sd:Chained {cvt:DifferentValuesToNull}, {sd:ToDouble}}, FallbackValue=1}"
-                           DecimalPlaces="{Binding DecimalPlaces, Converter={sd:Chained {cvt:DifferentValuesToNull}, {sd:ToDouble}}, FallbackValue=3}"
-                           WatermarkContentTemplate="{StaticResource DifferentValuesWatermarkTemplate}"
-                           DisplayUpDownButtons="False">
-        <i:Interaction.Behaviors>
-          <!--<sd:NumericTextBoxTransactionalRepeatButtonsBehavior UndoRedoService="{Binding Session.UndoRedoService}"/>-->
-          <behaviors:TextBoxPropertyValueValidationBehavior AdornerStoryboard="{StaticResource HighlightBorderAdornerValidationErrorStoryboard}"/>
-        </i:Interaction.Behaviors>
-      </sd:NumericTextBox>
+      <sd:TimeSpanEditor x:Name="TimeSpanEditor" Margin="2"
+                         Value="{Binding NodeValue, Converter={cvt:DifferentValuesToNull}}"
+                         WatermarkContentTemplate="{StaticResource DifferentValuesWatermarkTemplate}">
+      </sd:TimeSpanEditor>
       <DataTemplate.Triggers>
         <DataTrigger Binding="{Binding NodeValue}" Value="{x:Static qvm:NodeViewModel.DifferentValues}">
-          <Setter TargetName="TextBox" Property="WatermarkContent" Value="{sd:Localize (Different values)}"/>
+          <Setter TargetName="TimeSpanEditor" Property="WatermarkContent" Value="×"/>
+          <Setter TargetName="TimeSpanEditor" Property="IsEnabled" Value="False"/>
         </DataTrigger>
       </DataTemplate.Triggers>
     </DataTemplate>
   </view:TypeMatchTemplateProvider>
 
+  <!-- Provider for date time editor -->
+  <view:TypeMatchTemplateProvider x:Key="DateTimePropertyTemplateProvider" Type="{x:Type s:DateTime}" edvw:PropertyViewHelper.TemplateCategory="PropertyEditor">
+    <DataTemplate DataType="qvm:NodeViewModel">
+      <sd:DateTimeEditor x:Name="DateTimeEditor" Margin="2"
+                         Value="{Binding NodeValue, Converter={cvt:DifferentValuesToNull}}"
+                         WatermarkContentTemplate="{StaticResource DifferentValuesWatermarkTemplate}">
+      </sd:DateTimeEditor>
+      <DataTemplate.Triggers>
+        <DataTrigger Binding="{Binding NodeValue}" Value="{x:Static qvm:NodeViewModel.DifferentValues}">
+          <Setter TargetName="DateTimeEditor" Property="WatermarkContent" Value="{sd:Localize (Different values)}"/>
```

Comment:
What do you mean here by `Different values`? When would this show up?

Reply:
When you select multiple objects containing the same script. If the values differ, this string will be shown.